vmdb/app/models/miq_provision/options_helper.rb

```diff
@@ -26,7 +26,7 @@ def get_source
     ems = source.ext_management_system
     raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is not attached to a Management System" if ems.nil?
     raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> that does not support Provisioning" unless MiqProvision::SUPPORTED_EMS_CLASSES.include?(ems.class.name)
-    raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> with invalid credentials" if ems.authentication_invalid?
+    raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> with invalid credentials" if ems.missing_credentials?
```

Comment:
@jrafanie All callers still have to deal with conditions when connection might fail, because even if it says the status is ok, at the moment of connect it may not be.  So, I think we should only use authentication_status_ok as a pre-check for heavy operations...like starting the workers...don't start them unless we have known-good creds.  The worker still should expect a failure and deal with it accordingly (likely by setting the known-good status to bad).

This one, I think we should fail on known-bad connections, but the code @gmcculloug has for dealing with bad connections needs to stay (but perhaps should be updated to set the connection as bad if we *know* it's bad)